src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int DEFAULT_SIZE = 0;
+    private static final int SIMPLE_NUMBER_FOR_HASHCODE = 17;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Object[] table = new Object[DEFAULT_INITIAL_CAPACITY];
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
-
+        checkTableSize();
+        addNodeToTable(key, value);
     }
 
     @Override
     public V getValue(K key) {
+        int keyHashCode = getHashCode(key);
+        int indexOfBucket = Math.abs(findBucket(keyHashCode));
+        if (table[indexOfBucket] != null) {
+            Node<K, V> currentNode = (Node<K, V>) table[indexOfBucket];
+            while (true) {
+                if (Objects.equals(currentNode.key, key)) {
+                    return currentNode.value;
+                }
+                currentNode = currentNode.next != null ? currentNode.next : null;
```

Comment:
if you use `currentNode != null` as while-loop condition you can avoid the ternary operator here. But it seems to me that it will be better, maybe I'm wrong.